cmd/notary/cert.go

```diff
@@ -159,6 +166,51 @@ func (c *certCommander) certRemove(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+// certRotate replaces a certificate with a new version
+func (c *certCommander) certRotate(cmd *cobra.Command, args []string) error {
```

Comment:
It seems to try to rotate all the root keys - if we don't have the private key to any of them, it seems to fail.  There probably shouldn't be like 50 root keys on a repo, so it seems like just rotating all the ones for which we hold the key seems ok?  On the other hand, since there should be so few keys on the repo, maybe this command should take the fingerprint instead?

We might want to alter `cert list` to also look at the `root.json`s, instead of just the certs on disk, and to star whichever ones we have private keys for?  That way users can figure out which cert they want to rotate.